docs/docs/documentation/getting-started/installation/backend-config.md

```diff
@@ -16,6 +16,7 @@
 | API_DOCS                      |         True          | Turns on/off access to the API documentation locally                                               |
 | TZ                            |          UTC          | Must be set to get correct date/time on the server                                                 |
 | ALLOW_SIGNUP<super>\*</super> |         false         | Allow user sign-up without token                                                                   |
+| ALLOW_PASSWORD_LOGIN          |         true          | Whether or not to display the username+password input fields                                       |
```

Comment:
Maybe add a quick "why?" to this. Something like "Keep this on unless using a third-party authentication service (such as OIDC)."

Reply:
Good idea, thanks!